pallets/parachain-staking/Cargo.toml

```diff
@@ -0,0 +1,49 @@
+[package]
```

Comment:
We need to prefix the pallet name as `pallet-*`, otherwise some benchmarking scripts won't work.
Also the author and version adjustment please

Reply:
done.